docs/DeveloperGuide.md

```diff
@@ -1,38 +1,219 @@
+
+### Legend
+
+| Symbol         | Meaning                                                     |
+| -------------- |------------------------------------------------------------ |
+| ❗ IMPORTANT   | These are important instructions that you should follow.   |
+| ✏ Note   | These are important details that you should take note of.   |
+| ⬆ Back to top  | Click to scroll back up to the `Table of Contents`.         |
+| 🐵 🦊 🦁      | Animals indicate you have reached a new section.            |
+
+[⬆ Back to top](https://ay2324s1-cs2113t-w11-2.github.io/tp/DeveloperGuide.html#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------------------------
+
+## Quick Start 🐵
+
+1. Ensure you have Java 11 or above installed in your Computer. (What is my [Java version](https://www.java.com/en/download/help/version_manual.html)?)
+   
+2. Download the latest `WildWatch.jar` from [here](https://github.com/AY2324S1-CS2113T-W11-2/tp/releases).
+
+3. Copy the file to the folder you want to use as the home folder for your WildWatch.
+
+4. Open a command terminal, `cd` into the folder you put the jar file in, and run the following command: `java -jar WildWatch.jar`.
+You should see the welcome screen as the diagram below.
+
+5. Type commands beside `>>>` below the horizontal line and press `Enter` to execute it. Some example commands you can try:
+- `help` : Shows the help page.  
+- `bye` : Exits the program.  
+
+```
+____________________________________________________________
+____    __    ____  __   __       _______
+\   \  /  \  /   / |  | |  |     |       \
+ \   \/    \/   /  |  | |  |     |  .--.  |
+  \            /   |  | |  |     |  |  |  |
+   \    /\    /    |  | |  `----.|  '--'  |
+    \__/  \__/     |__| |_______||_______/
+____    __    ____  ___   .___________.  ______  __    __
+\   \  /  \  /   / /   \  |           | /      ||  |  |  |
+ \   \/    \/   / /  ^  \ `---|  |----`|  ,----'|  |__|  |
+  \            / /  /_\  \    |  |     |  |     |   __   |
+   \    /\    / /  _____  \   |  |     |  `----.|  |  |  |
+    \__/  \__/ /__/     \__\  |__|      \______||__|  |__|
+____________________________________________________________
+Hello there! Welcome to WildWatch!
+
+Checking if "WildWatch.txt" already exists...
+File does not exist.
+Creating new file...
+File created successfully.
+What would you like to do?
+____________________________________________________________
+>>> 
+```
+
+[⬆ Back to top](https://ay2324s1-cs2113t-w11-2.github.io/tp/DeveloperGuide.html#table-of-contents)
+
+--------------------------------------------------------------------------------------------------------------------------------------
+
+## Design & Implementation
+
+### System Architecture
+
+![System Architecture](https://github.com/AY2324S1-CS2113T-W11-2/tp/assets/69474977/90309a3c-f784-4ffb-8eef-65735c05ec52)
```

Comment:
<img width="804" alt="image" src="https://github.com/nus-cs2113-AY2324S1/tp/assets/97169613/86ba4b21-26dc-41cc-a5ff-e72619e304ae">
Perhaps you could remove the class names below as it is already mentioned above.